### PR TITLE
Update references to measurementlab throughout

### PR DESCRIPTION
**Pull request by stephen-soltesz**
This change updates all remaining references to "measurement kit", "measurement_kit" or "MEASUREMENT_KIT" to variations of "measurement lab". This change also updates the "single_include" version of `libndt7.hpp` and what I hope will be a final update to the README.md taking all recent changes into account.

Part of:
* https://github.com/m-lab/ndt7-client-cc/issues/2

- - -
This change is [Reviewable](https://reviewable.io/reviews/m-lab/ndt7-client-cc/11)

### CODE REVIEW

File: include/libndt7/internal/curlx.hpp [hunk 1]
```diff
@@ -1,8 +1,8 @@
-// Part of Measurement Kit <https://measurement-kit.github.io/>.
-// Measurement Kit is free software under the BSD license. See AUTHORS
+// Part of Measurement Lab <https://www.measurementlab.net/>.
+// Measurement Lab libndt7 is free software under the BSD license. See AUTHORS
 // and LICENSE for more information on the copying conditions.
-#ifndef MEASUREMENT_KIT_LIBNDT7_INTERNAL_CURLX_HPP
-#define MEASUREMENT_KIT_LIBNDT7_INTERNAL_CURLX_HPP
+#ifndef measurementlab_LIBNDT7_INTERNAL_CURLX_HPP
+#define measurementlab_LIBNDT7_INTERNAL_CURLX_HPP
```
Comment (bassosimone) [inline include/libndt7/internal/curlx.hpp:5]:
```suggestion
#ifndef MEASUREMENTLAB_LIBNDT7_INTERNAL_CURLX_HPP
#define MEASUREMENTLAB_LIBNDT7_INTERNAL_CURLX_HPP
```

File: include/libndt7/internal/logger.hpp [hunk 2]
```diff
@@ -1,15 +1,15 @@
-// Part of Measurement Kit <https://measurement-kit.github.io/>.
-// Measurement Kit is free software under the BSD license. See AUTHORS
+// Part of Measurement Lab <https://www.measurementlab.net/>.
+// Measurement Lab libndt7 is free software under the BSD license. See AUTHORS
 // and LICENSE for more information on the copying conditions.
-#ifndef MEASUREMENT_KIT_LIBNDT7_INTERNAL_LOGGER_HPP
-#define MEASUREMENT_KIT_LIBNDT7_INTERNAL_LOGGER_HPP
+#ifndef measurementlab_LIBNDT7_INTERNAL_LOGGER_HPP
+#define measurementlab_LIBNDT7_INTERNAL_LOGGER_HPP
```
Comment (bassosimone) [inline include/libndt7/internal/logger.hpp:5]:
```suggestion
#ifndef MEASUREMENTLAB_LIBNDT7_INTERNAL_LOGGER_HPP
#define MEASUREMENTLAB_LIBNDT7_INTERNAL_LOGGER_HPP
```

File: README.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 [![GitHub license](https://img.shields.io/github/license/m-lab/ndt7-client-cc.svg)](https://raw.githubusercontent.com/m-lab/ndt7-client-cc/main/LICENSE) [![Github Releases](https://img.shields.io/github/release/m-lab/ndt7-client-cc.svg)](https://github.com/m-lab/ndt7-client-cc/releases) [![Build Status](https://app.travis-ci.com/m-lab/ndt7-client-cc.svg?branch=main)](https://app.travis-ci.com/m-lab/ndt7-client-cc) [![codecov](https://codecov.io/gh/m-lab/ndt7-client-cc/branch/main/graph/badge.svg)](https://codecov.io/gh/m-lab/ndt7-client-cc)
 
-**Note**: this project is currently unmaintained due to lack of time. Maybe
-one day we'll find a fix for this issue and resume development...
+**Note**: This project is community supported and provided as-is for users and
+client integrators. Contributions are welcome (PRs and issues).
 
 ![tardis](docs/tardis.gif
   "It's not supposed to make that noise. You leave the brakes on.")
```
Comment (bassosimone) [inline README.md:9]: I don't know if you want to keep the GIF or not. My original intention for the GIF, which depicts a time traveling machine, was to say that, had I had time, I had worked happily on the project but I could not.

**Review by bassosimone — APPROVED** [review]
LGTM but I suggested editing the header guards because in a couple of cases you are using lowercase `measurementlab` rather than `MEASUREMENTLAB`. I did not provide suggestions for the single_include as well, since you can regenerate it easily using cmake. Also, I suggested that you may want to remove the Tardis reference from the readme as well as the associated GIF.

**Review by stephen-soltesz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/ndt7-client-cc/11)* status: <span>:shipit:</span> complete! 1 of 1 approvals obtained (waiting on @bassosimone)

___
*[`README.md` line 9 at r1](https://reviewable.io/reviews/m-lab/ndt7-client-cc/11#gh-1433769366:-NmC5NkUAQ572PugrVyN:bss4a2b) ([raw file](https://github.com/m-lab/ndt7-client-cc/blob/577eaf477beaa64ce64acf3de03dc6143f8c87d7/README.md#L9)):*
<details><summary><i>Previously, bassosimone (Simone Basso) wrote…</i></summary><blockquote>

I don't know if you want to keep the GIF or not. My original intention for the GIF, which depicts a time traveling machine, was to say that, had I had time, I had worked happily on the project but I could not.
</blockquote></details>

Removed.
___
*[`include/libndt7/internal/curlx.hpp` line 5 at r1](https://reviewable.io/reviews/m-lab/ndt7-client-cc/11#gh-1433764624:-NmC5K6Z1fsM9YaYXhYO:bo2d0mh) ([raw file](https://github.com/m-lab/ndt7-client-cc/blob/577eaf477beaa64ce64acf3de03dc6143f8c87d7/include/libndt7/internal/curlx.hpp#L5)):*
<details><summary><i>Previously, bassosimone (Simone Basso) wrote…</i></summary><blockquote>

```suggestion
#ifndef MEASUREMENTLAB_LIBNDT7_INTERNAL_CURLX_HPP
#define MEASUREMENTLAB_LIBNDT7_INTERNAL_CURLX_HPP
```
</blockquote></details>

Thank you for catching these. Fixed.
___
*[`include/libndt7/internal/logger.hpp` line 5 at r1](https://reviewable.io/reviews/m-lab/ndt7-client-cc/11#gh-1433765376:-NmC5MCi7jFrJbNzjODr:bo2d0mh) ([raw file](https://github.com/m-lab/ndt7-client-cc/blob/577eaf477beaa64ce64acf3de03dc6143f8c87d7/include/libndt7/internal/logger.hpp#L5)):*
<details><summary><i>Previously, bassosimone (Simone Basso) wrote…</i></summary><blockquote>

```suggestion
#ifndef MEASUREMENTLAB_LIBNDT7_INTERNAL_LOGGER_HPP
#define MEASUREMENTLAB_LIBNDT7_INTERNAL_LOGGER_HPP
```
</blockquote></details>

Thank you for catching these. Fixed.

**Comment by stephen-soltesz** [comment]
Thank you!

**Pull request merged**